conserver/main.c

```diff
@@ -53,8 +53,8 @@ int fAll = 0, fNoinit = 0, fVersion = 0, fStrip = 0, fReopen =
 char *pcConfig = CONFIGFILE;
 int cMaxMemb = MAXMEMB;
 #if USE_IPV6
-struct addrinfo *bindAddr;
-struct addrinfo *bindBaseAddr;
+struct addrinfo *bindAddr = (struct addrinfo *)0;
+struct addrinfo *bindBaseAddr = (struct addrinfo *)0;
```

Comment:
This change is unnecessary.  These are globals, and have "static storage duration" and thus are already guaranteed to be initialized to NULL.

In section 3.5.7 of the C89 drafts available onliine:
"If an object that has static storage duration is not initialized explicitly, it is initialized implicitly as if every member that has arithmetic type were assigned 0 and every member that has pointer type were assigned a null pointer constant. "